Arm/Insts/Cosim/disasm.sh

```diff
@@ -18,7 +20,7 @@ if [ -e instance.o ]; then
 
     # Do not print the address -- just the instruction bytes and its
     # corresponding disassembly.
-    snip_inst="awk '{\$1=\"\"; print}'"
+    snip_inst="awk '{\$2=\"\"; print}'"
```

Comment:
```suggestion
    snip_inst="awk '{\$1=\"\"; print}'"
```
Why `$2`? We'd like to see the 32-bit instruction value in the output, not the offset.

Reply:
@shigoel it needs to be `$2` because we now take an additional argument which is the file name to read from (`INFILE`), no?

Reply:
Fixed, d'oh, it's an `awk` $1 not a shell `$1`